content/docs/contributing/blog.md

```diff
@@ -1,7 +1,7 @@
 # Writing Blog Posts
 
-To create a new post, you need to create a PR to the DVC website
-[project](https://github.com/iterative/dvc.org).
+To create a new post, you need to create a PR to the Iterative.ai website
```

Comment:
Unfortunately, It won't work (iterative.ai repo is private). Let's just mention that please concat us? For internal users, I think we should have some docs in the iterative.ai/README

Reply:
Ohhh, right. I completely missed that. 
I'll close this one and follow-up with another one.

Reply:
My feeling at the moment is we don't have the capacity to take these requests so leave it out altogether. We haven't accepted a blog post from an outside source to our blog in years, so I don't see the point of contacting us. 

[We have instructions for blog writing here](https://www.notion.so/iterative/SOP-Blog-Review-Process-0784e7d5519249aa8f915906156c6882).  It needs to be refined [with @casperdcl's suggestions here.](https://iterativeai.slack.com/archives/C6YHPP2TB/p1668103968038969). But this may be too much for a README which should just have the how-to internally with a link to the review process.